tests/ClusterControllerTests.txt

```diff
@@ -0,0 +1,7 @@
+testTitle=UnitTests
+startDelay=0
```

Comment:
This test won't run in Joshua, unless it's placed inside a folder, say fast, under `tests` folder.